pkg/serverconfig/types.go

```diff
@@ -58,3 +59,11 @@ type Customization struct {
 type Providers struct {
 	StatuspageID string `yaml:"statuspageID,omitempty"`
 }
+
+type HelmChartRepo struct {
+	Url string `yaml:"url,omitempty"`
```

Comment:
Am I mistaken, or does this not match the example? It looks like this would be...

```yaml
helm:
  chart-repo:
    url: https://example.com/helm
```

Reply:
Yes, my mistake, it should be as you stated - I have updated the example above.

Reply:
In that case I'd suggest:

```yaml
html:
  chartRepository:
    url: https://example.com/helm
    caFile: /path/to/ca.crt
```

Reply:
fixed.